### ISSUE TITLE
MacOS Install instructions don't work

### ISSUE THREAD

**Issue by rick-rheo** [emscripten-core/emscripten#16230]
Please include the following in your bug report:

**Version of emscripten/emsdk:**
Can't run emcc so can't give you this

Following the instuctions from https://emscripten.org/docs/getting_started/downloads.html  on MacOS  does not work.  THere's no errors or anything  but   there's no emcc  anywhere.   This is my history

513  git clone https://github.com/emscripten-core/emsdk.git
514  cd emsdk
515  ./emsdk install latest
516  ./emsdk activate latest
517  source ./emsdk_env.sh
518  ./emcc -v

last one says   -bash: ./emcc: No such file or directory

There is no emcc  file anywhere on my hard drive.

I have latest XCode and Python etc.

**Comment by sbc100** [comment]
In this example you don't want the "./" at the start of "./emcc".   Once its in your PATH then you can just run `emcc -v` without the leading "./".

I don't see anything on https://emscripten.org/docs/getting_started/downloads.html that shows the leading `./`.. maybe I'm missing something?

**Comment by rick-rheo** [comment]
I'm not sure what you're looking at but that page most definitely has the leading ./ in the instructions

![image](https://user-images.githubusercontent.com/26754458/154508052-59de09bb-d80b-4cf3-8e88-63b04c8bc93a.png)

...and after running the commands in the instructions  my path has not been changed and there's not instructions to manually edit the path.

**Comment by sbc100** [comment]
`source ./emsdk_env.sh` should change your path.   After you run this, you should have emcc in your path (try running `emcc -v` and `which emcc`).

It looks like you are refering to the Tutorial at https://emscripten.org/docs/getting_started/Tutorial.html..  I think perhaps we should update it such that the `./` is not used.  The `./` prefix is for folks that are using the git checkout of emscripten (and therefore emcc is in the current directory).

**Comment by rick-rheo** [comment]
It doesn't.  I ran in one two machines. My path is not changed on either.

**Comment by sbc100** [comment]
Does `source ./emsdk_env.sh` output anything at all?

What does `which emcc` report?

**Comment by sbc100** [comment]
Can you show the full output of ` ./emsdk install latest` and `./emsdk active latest` and `source ./emsdk_env.sh`?

**Comment by rick-rheo** [comment]
It says its setting the path and indeed does for that instance of a terminal but opening when another terminal the path is unchanged.

`Adding directories to PATH:
PATH += /Users/rick/dev/webasm/emsdk
PATH += /Users/rick/dev/webasm/emsdk/upstream/emscripten
PATH += /Users/rick/dev/webasm/emsdk/node/14.18.2_64bit/bin

Setting environment variables:
PATH = /Users/rick/dev/webasm/emsdk:/Users/rick/dev/webasm/emsdk/upstream/emscripten:/Users/rick/dev/webasm/emsdk/node/14.18.2_64bit/bin:/usr/local/opt/mysql@5.7/bin:~/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/go/bin:/opt/X11/bin:/Library/Apple/usr/bin:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/Applications/AraxisMerge.app/Contents/Utilities:/Users/rick/bin
EMSDK = /Users/rick/dev/webasm/emsdk
EM_CONFIG = /Users/rick/dev/webasm/emsdk/.emscripten
EMSDK_NODE = /Users/rick/dev/webasm/emsdk/node/14.18.2_64bit/bin/node
EMSDK_PYTHON = /Users/rick/dev/webasm/emsdk/python/3.9.2_64bit/bin/python3
SSL_CERT_FILE = /Users/rick/dev/webasm/emsdk/python/3.9.2_64bit/lib/python3.9/site-packages/certifi/cacert.pem
emsdk[main]$ which emcc
/Users/rick/dev/webasm/emsdk/upstream/emscripten/emcc`

**Comment by rick-rheo** [comment]
Maybe this video will help clarify the problem

https://recordit.co/WBSq4NzQJN

**Comment by rick-rheo** [comment]
Right, OK but it would be helpful to have that in the instructions.

**Comment by sbc100** [comment]
I believe `emsdk activate` will tell you about this, but yes if its not in the emsdk docs it certainly should be.

This is the message that `emsdk activate` prints which is supposed to let you know what to do next:

```
Setting the following tools as active:
   node-14.18.2-64bit
   releases-upstream-0e2a01a034a4d5c4a13359931b2435f43901de85-64bit

Next steps:
- To conveniently access emsdk tools from the command line,
  consider adding the following directories to your PATH:
    /home/sbc/dev/wasm/emsdk
    /home/sbc/dev/wasm/emsdk/node/14.18.2_64bit/bin
    /home/sbc/dev/wasm/emsdk/upstream/emscripten
- This can be done for the current shell by running:
    source "/home/sbc/dev/wasm/emsdk/emsdk_env.sh"
- Configure emsdk in your shell startup scripts by running:
    echo 'source "/home/sbc/dev/wasm/emsdk/emsdk_env.sh"' >> $HOME/.bash_profile
```

**Comment by JarlenJohn** [comment]
> I believe `emsdk activate` will tell you about this, but yes if its not in the emsdk docs it certainly should be.
> 
> This is the message that `emsdk activate` prints which is supposed to let you know what to do next:
> 
> ```
> Setting the following tools as active:
>    node-14.18.2-64bit
>    releases-upstream-0e2a01a034a4d5c4a13359931b2435f43901de85-64bit
> 
> Next steps:
> - To conveniently access emsdk tools from the command line,
>   consider adding the following directories to your PATH:
>     /home/sbc/dev/wasm/emsdk
>     /home/sbc/dev/wasm/emsdk/node/14.18.2_64bit/bin
>     /home/sbc/dev/wasm/emsdk/upstream/emscripten
> - This can be done for the current shell by running:
>     source "/home/sbc/dev/wasm/emsdk/emsdk_env.sh"
> - Configure emsdk in your shell startup scripts by running:
>     echo 'source "/home/sbc/dev/wasm/emsdk/emsdk_env.sh"' >> $HOME/.bash_profile
> ```

I meet same log, what can I do? 

`emsdk activate node-14.18.2-64bit `?    it print log same as before. 
Thank you in advance！

**Comment by sbc100** [comment]
@JarlenJohn can you explain what the issue your are facing is?  

That log message is supposed to be printed.  You should be able to run `source emsdk_env.sh` in your current shell to start using emsdk.
